src/cmd-buildupload

```diff
@@ -35,8 +35,6 @@ def parse_args():
     parser.add_argument("--dry-run", help="Just print and exit",
                         action='store_true')
     group = parser.add_mutually_exclusive_group()
```

Comment:
We can drop this mutually-exclusive group too now, right?

Reply:
Hm yes, but I plan to re-add another variant there soon...